src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterOptions.cs

```diff
@@ -93,9 +135,11 @@ public bool SkipValidation
             }
         }
 
-        internal bool IndentedOrNotSkipValidation => _optionsMask != SkipValidationBit; // Equivalent to: Indented || !SkipValidation;
+        internal bool IndentedOrNotSkipValidation => Indented || !SkipValidation;
```

Comment:
Not sure if this is being used in the hot path, but could it be expressed using bitwise logic instead of a logical or?

Reply:
It is used for any start and end of an object or array.
Move to bitwise logic by applying a mask on first 2 bits.